monitor/main.py

```diff
@@ -86,9 +94,29 @@ def step_number_to_timestamp(step):
 
 
 class AppStateV1(NamedTuple):
-    block_fetcher_state: Any
-    skip_reporter_state: Any
-    offline_reporter_state: Any
+    block_fetcher_state: BlockFetcherStateV1
+    skip_reporter_state: SkipReporterState
+    offline_reporter_state: OfflineReporterStateV1
+
+
+class AppStateV2(NamedTuple):
+    block_fetcher_state: BlockFetcherStateV1
+    skip_reporter_state: SkipReporterStateV2
+    offline_reporter_state: OfflineReporterStateV2
+
+
+def upgradeV1toV2(v1: AppStateV1):
```

Comment:
```suggestion
def upgrade_v1_to_v2(v1: AppStateV1) -> AppStateV2:
```

Don't see a good reason why to not use snake case here.

Reply:
woops, my bad, of course this should be snake case.